pandas/io/formats/csvs.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         doublequote: bool = True,
         escapechar: Optional[str] = None,
         decimal=".",
+        storage_options: Optional[Dict[str, Any]] = None,
```

Comment:
maybe add this to typing.py, e.g. StorageOptions=....

Reply:
Up to you

Reply:
I am leaving it for now, but can do as you suggest if you request it

Reply:
seeing as we are doing this in lots of places I would add it